app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -27,18 +29,22 @@ fun PartiesScreen(
 	navController: NavHostController,
 ) {
 	when (val state = viewModel.parties.value) {
-		is PartiesState.Loading -> {
+		is Loading -> {
 			PicoverLoader()
 		}
 
-		is PartiesState.Loaded -> {
+		is Loaded -> {
 			LoadedContent(
 				parties = state.parties,
 				openDetails = {
 					navController.navigate("partyDetails/$it")
 				},
 			)
 		}
+
+		is Error -> {
+			// To be implemented
```

Comment:
Would be nice to add `TODO` to easily find such things later.

Reply:
yes, forgot about -> extracted !